doc/03-Configuration.md

```diff
@@ -93,3 +105,19 @@ allowing to keep this information for longer with a smaller storage footprint.
 
 A duration string is a sequence of decimal numbers and a unit suffix, such as `"20s"`.
 Valid units are `"ms"`, `"s"`, `"m"` and `"h"`.
+
+### Galera Cluster
+
+Icinga DB might not function properly or even crash occasionally when it is used with [Galera cluster](https://mariadb.com/kb/en/what-is-mariadb-galera-cluster/) setups.
+To avoid such unexpected crashes and reading/writing inconsistent data, Icinga DB sets the [wsrep_sync_wait](https://mariadb.com/kb/en/galera-cluster-system-variables/#wsrep_sync_wait)
+system variable for all its database connections. Consequently, strict cluster-wide causality checks are enforced before
```

Comment:
That's a strange way of starting this section given that this PR is supposed to fix exactly that. The documentation should primarily describe the behavior of the new version and this still reads like it doesn't work properly.

I'd start something like "Icinga DB expects a more consistent behavior from its database than a Galera cluster provides by default. To accommodate for this, Icinga DB sets Icinga DB sets the `wsrep_sync_wait` system variable ..."